### DISCUSSION TITLE
Clarity Stepper-how can i make all steps expended by default !!

### DISCUSSION THREAD

**Discussion by HamzaMansouri7** [vmware/clarity#5663] [Q&A] [answered]
Hi devs ! 
i use Clarity with angular 9  and i want to make all stepper's steps opned or expended by default , 
so , i mean , when i click on the step title(like contact ) i can open that content without click on (next) 
are there any directive or custom CSS  classes kan help me to de that  ??   
thanks 
![github](https://user-images.githubusercontent.com/51540362/109330885-775f7100-785c-11eb-8126-998e22c02d3b.PNG)

**Comment by milesich** [comment, marked as answer]
Use Accordion instead of Stepper.

**Comment by bbogdanov** [comment]
What you are looking for in the stepper is not what stepper is meant to support. Like @milesich said - Accordion is a better solution for your needs. 

[Here](https://github.com/vmware/clarity/issues/5287) is a thread where you can find a way to expand all panels at once.

**Reply by HamzaMansouri7** [reply]
hey thanks  so much gays , i find a solution with initialStep decorator , it been a long time 
thanks again
